TEST.md

```diff
@@ -0,0 +1,35 @@
+
+# Codacy test
+
+Testing the different configs of
+<https://github.com/remarkjs/remark-lint/tree/master/packages/remark-lint-ordered-list-marker-value>
+
+The `.remarkrc` contains the following
+
+```json
+{
+    "plugins": [
+        ["lint-ordered-list-marker-value", "ordered"]
+    ]
+}
+```
+
+So it should give errors on the #Single and #One sections
+
+## Single
+
+3. first
+
+3. second
+
+## One
+
+1. first
+
+1. second
+
+## Ordered
+
+1. first
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-spacing] Extraneous new line after list item](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5500053)